docs/installation/instructions.rst

```diff
@@ -61,8 +66,11 @@ PyPI
    $ pip3 install pulpcore
 
 4. If the the server.yaml file isn't in the default location of `/etc/pulp/server.yaml`, set the
-   PULP_SETTINGS environment variable to tell pulp where to find you server.yaml file.
-   ``export PULP_SETTINGS={virtualenv}/lib/python3.5/site-packages/pulpcore/etc/pulp/server.yaml``
+   PULP_SETTINGS environment variable to tell pulp where to find you server.yaml file (note that
```

Comment:
s/pulp/Pulp/

Reply:
The parentheses contain a complete sentence, can the parens be removed and that turned into its own sentence with `Note`... ?